lib/pynput/keyboard/_xorg.py

```diff
@@ -112,14 +112,25 @@ def _from_media(cls, name, **kwargs):
         """
         return cls._from_symbol('XF86_Audio' + name, **kwargs)
 
+    @classmethod
+    def _from_ISO(cls, name, **kwargs):
+        """Creates a ISO key from a partial name.
+
+        :param str name: The name. The actual symbol name will be this string
+            with ``'ISO_'`` prepended.
+
+        :return: a key code
+        """
+        return cls._from_symbol('ISO_' + name, **kwargs)
+
 
 # pylint: disable=W0212
 class Key(enum.Enum):
     # Default keys
     alt = KeyCode._from_symbol('Alt_L')
     alt_l = KeyCode._from_symbol('Alt_L')
     alt_r = KeyCode._from_symbol('Alt_R')
-    alt_gr = KeyCode._from_symbol('Mode_switch')
+    alt_gr = KeyCode._from_symbol('ISO_Level3_Shift')
```

Comment:
Since you have added `KeyCode._from_ISO`, I think that would be appropriate to use here.

Reply:
Solved in https://github.com/moses-palmer/pynput/pull/401/commits/e90dc23e1fb541a9f9f3aa852c196f1fa86a9048